configurations/morcore2esuites/Configuration_TaakApiInt.xml

```diff
@@ -164,6 +124,12 @@
                 <Exit name="Reject" state="REJECTED"/>
             </Exits>
 
+        <SenderPipe name="Send2MorCore">
+            <HttpSender name="SendNotification" methodType="POST" url="/melding/{meldingUuid}/taakopdracht/{taakOpdrachtUuid}/notificatie"/>
```

Comment:
Where is it sending to? Currently it would probably send it to itself? If it needs to go to MoreCore, there is a root-url missing. Would need to add a DeploymentSpecifics property for the MorCore root-url and then reference it in the url like: url="${morcore.root-url}/melding/{meldingUuid}/taakopdracht/{taakOpdrachtUuid}/notificatie. 

I believe {meldingUuid} -> ${meldingUuid} and {taakOpdrachtUuid} -> ${taakOpdrachtUuid} to substitude from a sessionKey or param. I believe without the $ it would only work if you supply it with a param and use this syntax in a pattern.

Would also recommend to supply the url with a param instead of an attribute. That way the url is also visible in Ladybug, which helps a lot with debugging/troubleshooting.

Reply:
So yea this endpoint only needs to send a notification that a task changed, not the change itself just that there is a change. I don't know the root-url this is the url given in the documentation. What you are saying about the params is true I will add these. Also with the other senders. The documentation does not give a lot of information about the required params.